{"name": "Test get all orders failure due to database error", "status": "passed", "start": 1728544651542, "stop": 1728544651542, "uuid": "c9b13afb-4209-482d-95df-01699b66e471", "historyId": "3c9d9d194a213089a257334b78080fba", "testCaseId": "3c9d9d194a213089a257334b78080fba", "fullName": "tests.order_tests.TestCartService#test_get_all_orders_failure_due_to_db_error", "labels": [{"name": "story", "value": "Get All Orders Error"}, {"name": "epic", "value": "Marketplace"}, {"name": "feature", "value": "Cart Service"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "order_tests"}, {"name": "subSuite", "value": "TestCartService"}, {"name": "host", "value": "DESKTOP-GIQ7099"}, {"name": "thread", "value": "17540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.order_tests"}]}